adapters/flipp/flipp.go

```diff
@@ -231,7 +243,20 @@ func buildBid(decision *InlineModel, impId string) *openrtb2.Bid {
 		if decision.Contents[0].Data.Width != 0 {
 			bid.W = decision.Contents[0].Data.Width
 		}
-		bid.H = 0
+		customDataInterface := decision.Contents[0].Data.CustomData
+		customDataMap := customDataInterface.(map[string]interface{})
```

Comment:
Please add a test case where customdata is a different type to verify this adapter code gracefully handles and error and doesn't panic on an incorrect response.